test/livebook_web/live/settings_live_test.exs

```diff
@@ -0,0 +1,85 @@
+defmodule LivebookWeb.SettingsLiveTest do
+  use LivebookWeb.ConnCase, async: true
+  @moduletag :tmp_dir
+
+  import Phoenix.LiveViewTest
+
+  alias Livebook.Settings
+
+  setup %{tmp_dir: tmp_dir} do
+    Livebook.Storage.current().insert(:settings, "global", autosave_path: tmp_dir)
+
+    on_exit(fn ->
+      Livebook.Storage.current().insert(:settings, "global", autosave_path: nil)
+    end)
+
+    :ok
+  end
+
+  describe "environment variables configuration" do
+    setup do
+      on_exit(&Settings.clean_env_vars/0)
+      :ok
+    end
+
```

Comment:
Since this is `async: true` we don't want to clear the whole storage. We can remove specific env vars in test cases, but it is also fine if we don't remove all of them :)

```suggestion
```